http/server.go

```diff
@@ -272,7 +272,12 @@ func (h *handler) LatestRand(w http.ResponseWriter, r *http.Request) {
 	nextTime := time.Now()
 	if info != nil {
 		roundTime = time.Unix(chain.TimeOfRound(info.Period, info.GenesisTime, resp.Round()), 0)
-		nextTime = time.Unix(chain.TimeOfRound(info.Period, info.GenesisTime, resp.Round()+1), 0)
+		next := time.Unix(chain.TimeOfRound(info.Period, info.GenesisTime, resp.Round()+1), 0)
+		if next.After(nextTime) {
+			nextTime = next
+		} else {
+			nextTime = nextTime.Add(time.Second)
```

Comment:
I have mixed feelings with this... I'd rather set something like `info.Period/2`.. sure a new round might come before that but otherwise we are telling the CDN (main consumer of this info) to "please check again in 1 second" which is a bit too low.

Reply:
yeah, wasn't sure what the right fallback was. info.period/2 works for me